src/episode.py

```diff
@@ -0,0 +1,48 @@
+from telegram import Update
+import requests
+from bs4 import BeautifulSoup
+from telegram.ext.callbackcontext import CallbackContext
+import config
+
+# Define podcast formats
+urlAll = '/podcast/'
+urlNews = '/podcast/news/'
+urlLese = '/podcast/lesestunde/'
+urlWeg = '/podcast/der-weg/'
+urlInterviews = '/podcast/interviews/'
+
+def getEpisode(url: str) -> str:
+    """
+    Returns the link to the most recent episode or an error message, if
+    the request fails
+    """
+    try:
+        r = requests.get(url, timeout=5)
+        doc = BeautifulSoup(r.text, "html.parser")
+        return f"{config.EINUNDZWANZIG_URL + doc.select('.plain')[0].get('href')}"
+    except:
+        return "Es kann aktuell keine Verbindung zum Server aufgebaut werden. Schau doch solange auf Spotify vorbei: https://open.spotify.com/show/10408JFbE1n8MexfrBv33r"
+
+def episode(update: Update, context: CallbackContext):
+    """
+    Sends a link to the most recent podcast episode
+    """  
+    try:
+        format = str(context.args[0]).lower()
+    except: 
+        format = "alle"
+    
+    if format == "news":
+        message = getEpisode(config.EINUNDZWANZIG_URL + urlNews)
```

Comment:
I think it makes more sense to just provide the path of the url as the argument here, since the EINUNDZWANZIG_URL is always the same. It should just be added in the getEpisode function to avoid duplication.